src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -221,6 +249,145 @@ public static FamilyType ByName(string name)
             };
         }
 
+        /// <summary>
+        /// Create new Family Instance from Geometry (via SAT import)
```

Comment:
For the first point about creating FamilyType objects, now I understand from @kronz that the new requirement is to create FamilyType objects. But the comment still need to be updated.